### PR TITLE
Sample spring boot2

### PR DESCRIPTION
**Pull request by juergen-walter**
Only documentation minors

### CODE REVIEW

File: sample-spring-boot/README.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ See section [Features](#features) for details.
 ## Getting Started
 
 This sample application is contained in the Maven module sample-spring-boot.
-It can be built with a simple `mvn install`. 
 A minimal CF application manifest is provided in [manifest.yml.](manifest.yml)
+You should change the name before building the app to avoid naming conflicts.
```
Comment (KarstenSchnitter) [inline sample-spring-boot/README.md:12]: The documentation is written in passive without addressing the reader.

File: README.md [hunk 2]
```diff
@@ -307,9 +307,10 @@ description about how to apply this feature can be found
 Stacktraces can be logged within one log message. Further details can be found
 [here](https://github.com/SAP/cf-java-logging-support/wiki/Logging-Stack-Traces).
 
-## Sample Application
+## Sample Applications
 
-In order to illustrate how the different features are used, this repository includes a simple application in the  [./sample folder](./sample).
+In order to illustrate how the different features are used, this repository includes - a simple java application in the [./sample folder](./sample).
+- a simple Spring boot application in the [./sample-spring-boot folder](./sample-spring-boot).
```
Comment (KarstenSchnitter) [inline README.md:313]: Can you make that a proper list, not embedded in the text?

**Review by KarstenSchnitter — CHANGES_REQUESTED** [review]

File: sample-spring-boot/README.md (hunk 1 above)
Comment (juergen-walter) [reply sample-spring-boot/README.md]: pushed fix

File: README.md (hunk 2 above)
Comment (juergen-walter) [reply README.md]: pushed fix

**Comment by juergen-walter** [comment]
retest this please

**Comment by juergen-walter** [comment]
Changes have been made in the base branch or added as comments

**Pull request closed without merging**